src/data/dynamic/admin/Columns.js

```diff
@@ -27,6 +27,14 @@ export const generateAffiliation = (affiliations) => ({
     <Tag text={affiliations[getValue()]} color={COLORS[getValue()]} />
   ),
 });
+export const generatePanelist = (panelists) => ({
```

Comment:
this file is only for those values that get repeated, panels type is a singular thing so we don't need a function, it can be a column object directly